pkg/controller/installation/products/codeready/reconciler_test.go

```diff
@@ -2,19 +2,39 @@ package codeready
 
 import (
 	"errors"
+	chev1 "github.com/eclipse/che-operator/pkg/apis/org/v1"
 	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/marketplace"
+	operatorsv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	operatorsv1 "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"testing"
 
 	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/config"
 	k8sclient "k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/rest"
 
+	aerogearv1 "github.com/integr8ly/integreatly-operator/pkg/apis/aerogear/v1alpha1"
 	"github.com/sirupsen/logrus"
 	pkgclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	kafkav1 "github.com/integr8ly/integreatly-operator/pkg/apis/kafka.strimzi.io/v1alpha1"
+	marketplacev1 "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	corev1 "k8s.io/api/core/v1"
 )
 
 func TestCodeready(t *testing.T) {
+	testScheme := runtime.NewScheme()
```

Comment:
why do we need to do this?

Reply:
If we need to do this, feels like it should by in an init or separate method?

Reply:
yeah, this was a real arm ache, but without it the `client.Create` or `client.Get` etc would fail with unknown type.